sourcegraph/sourcegraph
# dave testing: pls ignore

- wip
- pass fsanitize flag to zig cc
- patch flag
- fix1
- Move patch to its right place
- Reflect updates in a few deps
- Revert cpp nullptr change
- Add a few comments
- revert unecessary changes
- Re-pin the deps, now I've reverted some changes
- Pin serde, due to issues with `rules_rust`
- Update snapshots
- Add a build setting forcing test runners toolchain
- fixup
- Tell mocha to use the right toolchain if needed
- Set select defaults to None
- fixup
- Make codeintel-qa work locally
- Make a test run to ensure I didn't break codeintel
- fixup
- Remove manual code-intel test
- Drop dev left-over
- Grammar/typo fixes



## Test plan
